doc/classes/PopupMenu.xml

```diff
@@ -45,6 +45,17 @@
 				[b]Note:[/b] Checkable items just display a checkmark, but don't have any built-in checking behavior and must be checked/unchecked manually. See [method set_item_checked] for more info on how to control it.
 			</description>
 		</method>
+		<method name="add_direct_submenu_item">
```

Comment:
Maybe `add_submenu_node_item`/`get_item_submenu_node`/`set_item_submenu_node`?

Reply:
idk, if you think it's fine then ok. Though "add node" might suggest that the method is always adding a node, while it can also reuse node if it's already added. I'm not sure what would be a better name.

Reply:
I'm just not sure what "direct" would signify here. I have no association with the word in this context. Like, I understand what you mean with it, but not sure it works for user facing API.

By the way, we don't have to make adding the node optional, since it's a new method. Don't really see when that would be needed.

Reply:
Mmh, true, but it also can "just work" if it's already a child of this popup menu?

Reply:
Ok I'll rename it.